vowpalwabbit/global_data.h

```diff
@@ -565,6 +565,7 @@ struct vw
   bool adaptive;            // Should I use adaptive individual learning rates?
   bool normalized_updates;  // Should every feature be normalized
   bool invariant_updates;   // Should we use importance aware/safe updates
+  uint32_t ftrl_size;
```

Comment:
Could we pass this as an argument to save_load instead?  That seems more elegant than sticking it in the global data structure.

Reply:
Done.

Reply:
Oops, I merged.   Should I revert, or do you want to do a separate pull request?

Reply:
Found the bug, not sure what best way to fix it, see mail